pkg/codegen/openapi/schema.go

```diff
@@ -371,8 +371,9 @@ func GenerateGoSchema(sref *base.SchemaProxy, path []string) (Schema, error) {
 			}
 
 			// We've got an object with some properties.
-			for _, pName := range SortedKeys(schema.Properties) {
-				p := schema.Properties.Value(pName)
+			for pair := schema.Properties.First(); pair != nil; pair = pair.Next() {
```

Comment:
Switching to this is a bit cheaper, I think. There's nothing wrong with the original code, though.